### PR TITLE
BaseCollection new id

### PR DESCRIPTION
**Pull request by msladek**
PR for the whole story
https://synjira.atlassian.net/browse/CELDEV-616

### CODE REVIEW

**Review by fpichler — COMMENTED** [review]
please create first a celements-subsystem-migration-manager version 3.0-SNAPSHOT. Otherwise this project fails after merging this PR.

**Comment by msladek** [comment]
@fpichler I'll will merge all Celements 3 related PRs in the correct order. please still approve if everything else is ok.

**Comment by msladek** [comment]
celements-subsystem-migration-manager is already on 3.0-SNAPSHOT

**Review by fpichler — APPROVED** [review]

**Review by ebeutler — APPROVED** [review]

**Pull request merged**
